pkgs/development/python-modules/patator/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, buildPythonPackage, isPy3k, python3Packages, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "patator";
+  version = "0.7";
+  name = "${pname}-${version}";
+  disabled = !(isPy3k);
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "335e432e6cc591437e316ba8c1da935484ca39fc79e595ccf60ccd9166e965f1";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
```

Comment:
Please add the dependencies as arguments instead of `python3Packages`.

Reply:
Why not. What's the difference ?